category/category.go

```diff
@@ -0,0 +1,255 @@
+package category
+
+import (
+	"context"
+	"strings"
+
+	"github.com/almighty/almighty-core/convert"
+	"github.com/almighty/almighty-core/errors"
+	"github.com/almighty/almighty-core/gormsupport"
+	"github.com/almighty/almighty-core/log"
+	"github.com/jinzhu/gorm"
+	errs "github.com/pkg/errors"
+	uuid "github.com/satori/go.uuid"
+)
+
+// Defines "type" string to be used while validating jsonapi spec based payload
+const (
+	APIStringType = "categories"
+)
+
+// String constants for system categories
+const (
+	PlannerRequirements = "planner.requirements"
+	PlannerIssues       = "planner.issues"
+)
+
+// Do not change these UUIDs!!!
```

Comment:
Three exclamation is too much :) Maybe you can add a `WARNING` tag before, so it becomes like this:
```
// WARNING: Do not change these UUIDs!
```